lib/internal/test_runner/test.js

```diff
@@ -1023,13 +1021,17 @@ class Test extends AsyncResource {
             threshold: this.config.functionCoverage, name: 'function' },
         ];
 
+        coverage.thresholds = { __proto__: null };
```

Comment:
Not a blocker, but would it make more sense to add this information to the `coverage` object in the coverage.js file where it is created. You should have access to all of the necessary information there, and that file is more dedicated to code coverage. I'm good with leaving the actual checking and reporting here though.